libshvcore/src/utils/shvjournalentry.cpp

```diff
@@ -16,6 +16,8 @@ const char *ShvJournalEntry::DOMAIN_SHV_SYSTEM = "SHV_SYS";
 const char* ShvJournalEntry::PATH_APP_START = "APP_START";
 //const char* ShvJournalEntry::PATH_SNAPSHOT_END = "SNAPSHOT_END";
 const char* ShvJournalEntry::PATH_DATA_MISSING = "DATA_MISSING";
+const char* ShvJournalEntry::PATH_DATA_DIRTY = "dirty";
```

Comment:
nebylo by lepsi `"DATA_DIRTY"`, kdyz uz mame `"DATA_MISSING"` ?

Reply:
Nechal jsem to kvuli kompatibilite, ale mozna mas pravdu, jeste to opravim.

Reply:
Opraveno